mmdeploy/codebase/base/mmcodebase.py

```diff
@@ -15,11 +16,11 @@ class MMCodebase(metaclass=ABCMeta):
 
     task_registry: Registry = None
 
-    def __init__() -> None:
+    def __init__(self) -> None:
         pass
 
     @classmethod
-    def get_task_class(cls, task: Task) -> type:
```

Comment:
Hi, I have a question here. The function will return a class. And the type of class is `type`. Is it ok to say the type of returned value is the class itself?